### DISCUSSION TITLE
DisableEnchants per world

### DISCUSSION THREAD

**Discussion by Rompepitas** [Zrips/CMI#8182] [Enhancement]
I would like the option to add a 'world' parameter to the `/disableenchant` command, as it would greatly assist in managing it. An example of the final command would be `/disableenchant <enchant> <world> (disable | enable)`, where if a world is not specified, it can be assumed that the command was executed for all worlds.

**Comment by CloudeLecaw** [comment]
you landet in the wrong area :-)

create your request here
https://github.com/Zrips/CMI/discussions/new?category=enhancement

![Screenshot_2023-10-13-06-40-47-850_com android chrome-edit](https://github.com/Zrips/CMI/assets/12080017/eeaa9804-3431-4cac-a52a-d422ba47dadb)
